### DISCUSSION TITLE
Can I make the content smaller of a pop-up card?

### DISCUSSION THREAD

**Discussion by telegrambeans** [Clooos/Bubble-Card#327] [Questions about config, custom styles and templates]
II'm looking for a way to make the content of the cards a bit smaller. On the tablet, the content is unfortunately displayed too large in landscape mode.

**Comment by Clooos** [comment]
Hi, you can find all the information you need here :
https://github.com/Clooos/Bubble-Card#styling

**Comment by telegrambeans** [comment]
Sorry, I'm a beginner in CSS style. How can i change the size of the cards? Do I have to use the font size?

**Comment by telegrambeans** [comment]
> Tut mir leid, ich bin ein Anfänger im CSS-Stil. Wie kann ich die Größe der Karten ändern? Muss ich die Schriftgröße verwenden?

Ich bräuchte dafür mal ein Beispiel. Ich probiere halt viel rum und möchte dabei lernen.

**Comment by MrBearPresident** [comment]
@telegrambeans have you figured this out? Can I close the discussion?
